### PR TITLE
Forms Automated Test - LPS-133113 | master

### PR DESCRIPTION
**Pull request by PauloVAS**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsRichTextField.testcase [hunk 1]
```diff
@@ -54,14 +54,58 @@ definition {
...
+			fieldType = "Rich Text");
+
+		FormFields.editHelpText(
+			fieldHelpText = "Help Text Test",
+			fieldName = "tip");
+
+		Form.gotoAdvancedTab();
+
+		CKEditor.addContentInWebContent(
+			content = "Rich Text Test",
```
Comment (CalumR23) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsRichTextField.testcase:81]: Hey @PauloVAS can you change this CKEditor macro to a Forms macro please?

Comment (PauloVAS) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsRichTextField.testcase:81]: Ok, starting now. Thanks

**Comment by PauloVAS** [comment]
@CalumR23, fixed!

**Comment by CalumR23** [comment]
ci:test:sf

**Comment by CalumR23** [comment]
ci:forward

**Comment by CalumR23** [comment]
ci:stop

**Comment by brittneyq** [comment]
Combined and sent in [103234](https://github.com/brianchandotcom/liferay-portal/pull/103234)

**Pull request closed without merging**